python/pyarrow/tests/test_orc.py

```diff
@@ -169,6 +171,36 @@ def test_orcfile_empty(datadir):
     assert table.schema == expected_schema
 
 
+def test_readwrite(tmpdir):
+    from pyarrow import orc
+    a = pa.array([1, None, 3, None])
+    b = pa.array([None, "Arrow", None, "ORC"])
+    table = pa.table({"int64": a, "utf8": b})
+    file = tmpdir.join("test.orc")
+    orc.write_table(table, file)
+    output_table = orc.read_table(file)
+    assert table.equals(output_table)
+
```

Comment:
Can you add one more read_table call here with eg `columns=["int64"]`, just to make sure that this keyword is properly passed through

Reply:
Sure!